app/code/community/Bolt/Boltpay/Helper/ConfigTrait.php

```diff
@@ -236,4 +236,38 @@ public static function getBoltPluginVersion()
 
         return null;
     }
+
+    /**
+     * Returns message to be displayed to customer if attempting to checkout with order lower than minimum amount
+     *
+     * @return string
+     */
+    public function getMinOrderDescriptionMessage()
+    {
+        return Mage::getStoreConfig('sales/minimum_order/description')
```

Comment:
Potential refactor here to:

return Mage::getStoreConfig('sales/minimum_order/description') ?:     Mage::helper('checkout')->__('Minimum order amount is %s',$this->getMinOrderAmountInStoreCurrency());

Not required. Just saw it as an opportunity.

Reply:
@johnmccombs1 
Can you make a PR with the change?